### DISCUSSION TITLE
Unable to run the abbreviation expansion program using spacy

### DISCUSSION THREAD

**Discussion by jithesh2** [explosion/spaCy#8206] [Help: Coding & Implementations] [answered]
I am using pycharm in windows 10 Environment with
spaCy version    3.0.0
Location         c:\users\jithe\anaconda3\lib\site-packages\spacy
Platform         Windows-10-10.0.19041-SP0
Python version   3.7.4
Pipelines        en_core_sci_sm (0.4.0), en_core_web_sm (3.0.0)
 
GEtting the following error

Traceback (most recent call last):
  File "C:/Users/jithe/PycharmProjects/scispacy-main/spacy.py", line 1, in <module>
    import spacy
  File "C:\Users\jithe\PycharmProjects\scispacy-main\spacy.py", line 2, in <module>
    from scispacy.abbreviation import AbbreviationDetector
  File "C:\Users\jithe\PycharmProjects\scispacy-main\scispacy\abbreviation.py", line 3, in <module>
    from spacy.tokens import Span, Doc
ModuleNotFoundError: No module named 'spacy.tokens'; 'spacy' is not a package

Kindly help

**Comment by polm** [comment, marked as answer]
Sorry to hear you're having trouble. However, ScispaCy is a separate project and not developed by us, in general you should ask at their repository for help. 

https://github.com/allenai/scispacy

That said, in this particular case I think the issue might be that your file is called `spacy.py`. Try changing it to something else like `spacy-test.py` and it should work.
